app/app.go

```diff
@@ -948,15 +941,7 @@ func (app *Evmos) BlockedAddrs() map[string]bool {
 		blockedAddrs[authtypes.NewModuleAddress(acc).String()] = true
 	}
 
-	blockedPrecompilesHex := []string{
-		p256precompile.PrecompileAddress,
-		bech32precompile.PrecompileAddress,
-		bankprecompile.PrecompileAddress,
-		stakingprecompile.PrecompileAddress,
-		distprecompile.PrecompileAddress,
-		ics20precompile.PrecompileAddress,
-		vestingprecompile.PrecompileAddress,
-	}
+	blockedPrecompilesHex := evmtypes.DefaultStaticPrecompiles
```

Comment:
please update also function docstring to reflect not only app's module can be blocked.